src/utils/setting_handler.py

```diff
@@ -46,32 +68,37 @@ def get_requestless_setting(setting_group, setting, journal):
     return setting
 
 
-def _get_setting(setting_group, setting, journal, lang, create, fallback):
+def _get_setting(
+        setting_group, setting, journal, lang, create, fallback, default=True):
+    if fallback:
+        _fallback = settings.LANGUAGE_CODE
+    else:
+        _fallback = None # deactivates fallback
+
     try:
-        setting = core_models.SettingValue.objects.language(lang).get(
-            setting__group__name=setting_group,
-            setting=setting,
-            journal=journal
+        return core_models.SettingValue.objects \
```

Comment:
Aren't backslashes the devil!? :P

Reply:
They are! but they are also a necessary devil in this case :hurtrealbad: